src/test/v2/__snapshots__/computation-and-formatting.test.ts.snap

```diff
@@ -27,21 +27,21 @@ Object {
       "stableBorrows": "0",
       "stableBorrowsETH": "0",
       "stableBorrowsUSD": "0",
-      "totalBorrows": "137635.694716770184532442",
-      "totalBorrowsETH": "224.903606951938320035",
-      "totalBorrowsUSD": "2249036069519383200.35",
```

Comment:
as you can see the numbers only changed in the late ecimals - for usd i fixed the testcae that's why the value is more off (https://github.com/aave/aave-js/pull/62/files#diff-07006b4d58b3fbb84f6ceab251c157f1adbfd20be6366ced86a8eb284d2c231eR78).
Yesterday marketmake participant reported that this doesn't work and it turned out he's been using '1670` or sth similar as ethprice so i guess at least in the tests we should have logical data.